include/fat.h

```diff
@@ -15,6 +15,17 @@
  */
 sblock_t find_first_free_block(const unsigned char *aes_key);
 
+/**
+ * @brief Find the first safe free block in the FAT table.
+ *
+ * @param aes_key The AES key to use for in place decryption of the FAT tables.
+ *
+ * @return sblock_t The index of the first free block, the difference with 
+ * "find_first_free_block" is that it only return a positive block index as it 
+ * initialize another FAT in case of out of range.
```

Comment:
À mettre dans `@brief` plutôt. Ne laisser que `@return sblock_t The index of the first free block`